### ISSUE TITLE
Material Input Component appearance

### ISSUE THREAD

**Issue by richehmer** [dart-lang/angular_components#311]
Apologies if this is off-base as I'm new to this, but I have 'outline' and 'fill' text input appearances in a design and am looking for a the best way to include them. It would be great to set the appearance of [Material Input](https://dart-lang.github.io/angular_components_example/#material_input) in the same way as [angular form fields](https://dart-lang.github.io/angular_components_example/#material_input)!

**Comment by nshahan** [comment]
Are you talking about the filled and outlined input fields as described in the material guidelines?
https://material.io/design/components/text-fields.html#filled-text-field
https://material.io/design/components/text-fields.html#outlined-text-field

If so, these were added to the guidelines at Google IO this year and we haven't yet rolled out the implementation for them. We expect to have a set of updated components that would include these, possibly by the end of the year. 

There are versions (not for AngularDart) available from the material components web team here: https://github.com/material-components/material-components-web

**Comment by richehmer** [comment]
Yes, that's what I'm referring to. I thought this issue might fall under 'widget request', but perhaps not.

**Comment by tauraamui** [comment]
Are there material input outline style options available for AngularDart by now?

**Comment by nshahan** [comment]
@tauraamui Not as part of this package yet.
